webpack.config.js

```diff
@@ -59,9 +59,7 @@ const webpackConfig = {
     new CopyWebpackPlugin([{from: 'static'}]),
     new webpack.IgnorePlugin(/canvas/),
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),  // Ignore all locale files of moment.js
-    new webpack.DefinePlugin({
-      'process.env': { NODE_ENV: JSON.stringify(NODE_ENV) }
-    })
+    new webpack.EnvironmentPlugin({ NODE_ENV: 'development', PARSER: null, FIREBASE_DB: null })
```

Comment:
Is there any way to start wordbots locally and overrides these currently?

Reply:
Yeah, you can do `PARSER=local yarn start`, `FIREBASE_DB=production yarn start`, etc.

Reply:
Oh I see... I didn't fully understand how this plugin differed from the previous one